README.md

```diff
@@ -273,6 +274,49 @@ your browser.
     'Client secret'.
 
 
+### 3.2. Facebook OAuth
+
+ 1. Go to https://developers.facebook.com/ and create an app.
+
+ 2. Add a Product 'Facebook Login' and select platform 'Web'. Enter 'Site URL' `https://{domain}/`.
+
+ 3. Go to 'Facebook Login' / 'Settings' and add 'Valid OAuth Redirect URIs':
+
+    	https://{domain}/accounts/facebook/login/callback/
+    	https://{domain}/sk/accounts/facebook/login/callback/
+    	https://{domain}/en/accounts/facebook/login/callback/
+
+    Where `{domain}` is your production domain or `127.0.0.1:8000` if in local development mode.
+    'Site URL' in form `127.0.0.1:8000` is invalid use `localhost:8000` instead.
+
+To use Facebook Login, you need to use https. Setup default protocol in `settings/common.py`.
+
+	ACCOUNT_DEFAULT_HTTP_PROTOCOL = u'https'
+
+To run a server with https you can use
+[RunServerPlus](https://django-extensions.readthedocs.org/en/latest/runserver_plus.html) from
+[django-extensions](https://github.com/django-extensions/django-extensions). To install run:
+
+	$ pip install django-extensions==1.0.3
+	$ pip install Werkzeug==0.16.1
+	$ pip install pyOpenSSL==19.1.0
+
+Add it to `INSTALLED_APPS` in `settings/common.py` file:
```

Comment:
@martinmacko47 nechat `settings/common.py` alebo pouzit iny settings-file?

Reply:
Asi skor `server_local.py`, kedze je to konfiguracia len ak bezis server na localhoste. V produkcii to netreba, https v produkcii sa riesi inac.

Reply:
:ok: